stories/qa/forms/textfield.stories.html

```diff
@@ -2,23 +2,23 @@
 	<h2>Sizes</h2>
 
 	<div class="form-field pr-u-marginBottom100">
-		<label class="formLabel" id="fieldSizlabel" for="fieldSiz">
+		<label class="formLabel" id="fieldSizelabel" for="fieldSize">
```

Comment:
Commenting to create another issue about matching the same pattern `id-topic` or `idTopic`:

- [fieldSizelabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR5)
- [fieldXSlabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR45).
- [fieldErrorEmptylabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR299)
- [fieldErrorFilledlabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR319)
- [fieldDisabledEmptylabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR351)
- [fieldDisabledFilledlabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR372)
- [fieldAlllabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR402)

The `(id|by)=".+[^-](label|message)"` regexp can help.